srtcore/utilities.h

```diff
@@ -140,6 +140,46 @@ written by
 #	define le64toh(x) letoh64(x)
 #endif
 
+#elif defined __sun
```

Comment:
`SUNOS` is defined in CMake in the case of Solaris. Might be better to use it instead of `__sun`.